consensus/service/src/byzantine_ledger/worker.rs

```diff
@@ -1251,6 +1251,69 @@ mod tests {
         assert_eq!(worker.pending_consensus_msgs.len(), 1);
     }
 
+    /// Should maintain the invariant that pending_values and pending_values map
```

Comment:
Looks like the pending_values and pending_values_map have been encapsulated in the PendingValues struct, so I think this comment is a little out of date. Maybe something like this is better:

```
/// Should maintain the invariant that pending_values only contains tx_hashes corresponding to transactions that are valid w.r.t. the current ledger.
```